chunky/src/java/se/llbit/chunky/renderer/RenderContext.java

```diff
@@ -66,47 +77,55 @@ public int numRenderThreads() {
   }
 
   /**
+   * @param descriptionFileName The name of the scene description file. NOTE: Do not include extension
```

Comment:
"The name of the scene description file, without the file extension". Don't yell at those who read the docs :smile: